sos/policies/redhat.py

```diff
@@ -412,13 +421,15 @@ class CentOsAtomicPolicy(RedHatAtomicPolicy):
     distro = "CentOS Atomic Host"
     vendor = "CentOS"
     vendor_url = "https://www.centos.org/"
+    id = "centos"
 
 
 class FedoraPolicy(RedHatPolicy):
 
     distro = "Fedora"
     vendor = "the Fedora Project"
     vendor_url = "https://fedoraproject.org/"
+    id = "fedora"
```

Comment:
Similarly, this would not be evaluated. First because `FedoraPolicy` subclasses `RedHatPolicy` and not `RHELPolicy` (intentional), and second because `FedoraPolicy` overrides `check()` and simply looks to see if `/etc/fedora-release` exists.

For Fedora, I'd say we don't ever need to check for ID, because the basic check is sufficient for all builds. If an admin is removing `/etc/fedora-release`, then there are likely major changes going on and at the very least we don't want to have sos misidentifying such a rebuild with the Fedora policy.

Reply:
I agree, my intend was that it would still work if for some reason Fedora would again rely on content of `/etc/os-release` (the generic code) instead of `/etc/fedora-release`.